modules/client-configuration/pages/contact-methods-intro.adoc

```diff
@@ -1,5 +1,5 @@
 [[contact-methods-intro]]
-= Contact Methods
+= Client Management Methods (Contact Methods)
```

Comment:
I woudld remove the (contact methods) bit. **Client Management Methods** seems like a good fit. Also keep in mind we should be swapping to sentence title syntax. 

Upcase only on the first item or proper names, etc.

**Client management methods**